src/components/ColorManager/index.tsx

```diff
@@ -0,0 +1,22 @@
+/** @jsx jsx */
+import * as React from "react";
+import { jsx, css } from "@emotion/core";
+import { Theme } from "../../types/global";
+import { setColorScheme } from "../../../lib/utils";
+
+interface Props {
+  colorScheme?: string;
+}
+
+const ColorManager: React.FC<Props> = ({ colorScheme = "dark", ...props }) => {
+  const scheme = setColorScheme(colorScheme);
+
+  const styles = ({ ...theme }: Theme) =>
+    css({
+      ...scheme,
+    });
+
+  return <main css={styles} {...props} />;
```

Comment:
we talked a bit about this in person but we should be doing less abstraction here and being more concise.